src/components/layout/Header.scss

```diff
@@ -50,4 +64,49 @@ header {
       color: $base_white;
     }
   }
+
+  &.medium {
+    .icon {
+      display: block;
+      padding: 0.5rem;
+    }
+
+    .menu {
+      background-color: $base_grey;
```

Comment:
Can we have a less "dirt-like" color ? Maybe black on white or white on black ...
The best option i can think of right now is black on white with a "blur" filter on the rest of the page to hide it (like a modal)